drivers/pppos.c

```diff
@@ -448,46 +477,21 @@ static void pppos_mainLoop(void* _state)
 			log_info("open success!");
 		}
 
-		serial_set_non_blocking(state->fd);
```

Comment:
In order to accomodate the different modems, I have put a few defines here and there. A `PPPOS_MODEM_TYPE` define is now required and it will break existing setups, but the flavor has to be somehow explicitly chosen anyway, be it in a build file or some other.

New defines now make it possible to merge the `lte430` branch with minimal work required.